packages/grid/_modules_/grid/components/menu/columnMenu/DefaultGridColumnHeaderMenuItems.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { ColumnsMenuItem } from './ColumnsMenuItem';
+import { FilterMenuItem } from './FilterMenuItem';
+import { GridColumnHeaderMenuItemProps } from './GridColumnHeaderMenu';
+import { HideColMenuItem } from './HideColMenuItem';
+import { SortMenuItems } from './SortMenuItems';
+
+export function DefaultGridColumnHeaderMenuItems({
+  hideMenu,
+  currentColumn,
+}: GridColumnHeaderMenuItemProps) {
+  return (
+    <React.Fragment>
```

Comment:
This can't work, the menu items need to be direct children of the MenuList, there can't be a fragment in between. It breaks the keyboard navigation

Reply:
I'm not sure I understand what you mean.
The keyboard navigation was not working in the menu before. 
And if I refactor to something that will not work, I don't see any benefit in the refactoring.
I think the keyboard navigation should be tackled in a separate issue.

Reply:
> The keyboard navigation was not working in the menu before.

True, I haven’t look at why. The only difference I could find with this working demo is the extra Fragment: https://material-ui.com/components/menus/#menu list-composition.

My main concern is that if we make this public, it would be better to not have to make a BC to fix the issue.

Reply:
The keyboard navigation is fixed in my second PR ;)